pkg/controller/controller.go

```diff
@@ -155,9 +154,8 @@ func (c *controller) Run(workers int, stopCh <-chan struct{}) {
 	for i := 0; i < workers; i++ {
```

Comment:
Besides having multiple queues processing instances differently, is it likely there is any problem with having multiple workers processing instances the same way?